### DISCUSSION TITLE
...solving for optimal weight factor : inf

### DISCUSSION THREAD

**Discussion by wkitlasten** [usgs/pestpp#238] [General]
` ...solving for optimal weight factor : inf`

That can't be good, right? Any suggestions on where to start looking for my mistakes?

**Comment by jtwhite79** [comment]
Im not sure its a mistake on your end @wkitlasten .  The dynamic regul solve in pestpp-glm needs an overhaul and I've not had time to work on it.  Can you get by with TSVD regularization?  Or maybe manually setting the prior information weights? Or using PEST_HP?

**Reply by wkitlasten** [reply]
My regularization was lazily added using `pyemu.helpers.zero_order_tikhonov(pst, parbounds=True)`, which seems to result in `wfinit 1.0`. Does that automatically trigger the dynamic regul solve? If so, is there a way to turn it off?

I checked and adjusted my reg weights. I am expecting reg phi to be about 1% of my measurement phi. I am getting `Starting regularization objective function : 0`, but I assume that is expected before any par adjustments.

I'm sure I'm missing something simple.

**Reply by wkitlasten** [reply]
I should also mention, I am trying to reuse my ies setup generated using pst_from with multiply and addition parameters. The initial value of the addition params is 0 (transform=none, parchglim=relative, offset=0), so maybe that is messing up the prior information equations/implementation?

**Comment by jtwhite79** [comment]
If you keep pestmode as estimation and leave out the regularization section, the pi equations should be treated just as any other observation.  

I'll try to run some experiments with pars that have a zero initial value...that's a good idea
